core/src/main/java/tc/oc/pgm/tntrender/TNTRenderMatchModule.java

```diff
@@ -101,4 +102,12 @@ private void updatePlayer(MatchPlayer player) {
       }
     }
   }
+
+  private static Location toBlockLocation(Location location) {
+    Location blockLoc = location.clone();
+    blockLoc.setX(blockLoc.getBlockX());
+    blockLoc.setY(blockLoc.getBlockY());
+    blockLoc.setZ(blockLoc.getBlockZ());
+    return blockLoc;
+  }
```

Comment:
If this is necessary because of multi-platform/version compat maybe add a commend about it?

Reply:
Yeah, it's needed for version compatibility. I can add a comment